packages/elements/src/slider/elements/slider.ts

```diff
@@ -438,6 +438,30 @@ export class Slider extends ControlElement {
     return markers;
   }
 
+  /**
+   * Finds the closest ancestor ef-slider-marker in the DOM tree.
+   * @param startingElement The HTML element to start searching from.
+   * @returns The found marker, or null if not found.
+   */
+  private findClosestMarker(startingElement: EventTarget | Node | null): SliderMarker | null {
```

Comment:
No need to accept `Node` here, right?
```suggestion
  private findClosestMarker(startingElement: EventTarget | null): SliderMarker | null {
```